app/src/main/java/de/danoeh/antennapod/fragment/AudioPlayerFragment.java

```diff
@@ -69,10 +70,10 @@
 public class AudioPlayerFragment extends Fragment implements
         ChapterSeekBar.OnSeekBarChangeListener, Toolbar.OnMenuItemClickListener {
     public static final String TAG = "AudioPlayerFragment";
-    private static final int POS_COVER = 0;
-    private static final int POS_SHOWNOTES = 1;
-    private static final int POS_CHAPTERS = 2;
-    private static final int NUM_CONTENT_FRAGMENTS = 3;
+    public static final int POS_COVER = 0;
+    public static final int POS_TABS = 1;
```

Comment:
Hmm i dont know because in EpisodeFragment theres also a position for the actual description fragment so i think we should use different names so not to get them confused..